spec/features/accessibility_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "accessibility", type: :feature, js: true do
+  context "when visiting home page" do
+    before do
+      visit "/"
+    end
+
+    it "complies with wcag" do
+      expect(page).to be_axe_clean
+        .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
+    end
+  end
+
+  context "when visiting computing page" do
+    before do
+      visit "/computing"
+    end
+
+    it "complies with wcag2aa wcag21a" do
+      expect(page).to be_axe_clean
+        .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
+    end
+  end
+
+  context "when visiting consultants page" do
+    before do
+      visit "/consultants"
+    end
+
+    it "complies with wcag2aa wcag21a" do
+      expect(page).to be_axe_clean
+        .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
+    end
+  end
+
+  context "when visiting specialists page" do
+    before do
+      visit "/specialists"
+    end
+
+    it "complies with wcag2aa wcag21a" do
+      expect(page).to be_axe_clean
+        .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
+    end
+  end
+
+  context "when visiting training page" do
+    before do
+      visit "/training"
+    end
+
+    it "complies with wcag2aa wcag21a" do
+      expect(page).to be_axe_clean
+        .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
```

Comment:
Let's remove `/consultants` and `/computing` since they redirect to the library production site. 

`/training` will fail with a routing error so let's remove it.